libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H757xx.py

```diff
@@ -1278,40 +1278,72 @@
 
 ADC1_map = {
     # format is PIN : ADC1_CHAN
-    "PF11"  :   2,
-    "PA6"	:	3,
-    "PC4"	:	4,
-    "PB1"	:	5,
-    "PF12"  :   6,
-    "PA7"	:	7,
-    "PC5"	:	8,
-    "PB0"	:	9,
+	"PA0_C"	:	0,
```

Comment:
I would do it without the _C, we normalise other pin names in these files as well, it varies based on which person in ST creates the particular datasheet

Reply:
The _C is the whole problem. Did you see the image in the description?

Reply:
The easiest thing to do is just remove support for those pins on the 240+25 pin count versions